<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArxWidget</class>
 <widget class="QWidget" name="ArxWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ARX - Wektory, Opóźnienie, Zakłócenia</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tabelaWektorow"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutRows">
     <item>
      <widget class="QPushButton" name="btnDodajWiersz">
       <property name="text">
        <string>Dodaj wiersz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinUsunWiersz">
       <property name="prefix">
        <string>Wiersz: </string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnUsunWiersz">
       <property name="text">
        <string>Usuń wiersz</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hLayoutParams">
     <item>
      <widget class="QLabel" name="labelOpoznienie">
       <property name="text">
        <string>Opóźnienie:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinOpoznienie">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelZaklocenia">
       <property name="text">
        <string>Zakłócenia:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spinZaklocenia">
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.010000000000000</double>
       </property>
       <property name="value">
        <double>0.010000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnZapisz">
     <property name="text">
      <string>Zapisz</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
